diff --git a/node_modules/react-native-paper/src/components/Menu/Menu.tsx b/node_modules/react-native-paper/src/components/Menu/Menu.tsx
index 55922c1..302cf7d 100644
--- a/node_modules/react-native-paper/src/components/Menu/Menu.tsx
+++ b/node_modules/react-native-paper/src/components/Menu/Menu.tsx
@@ -6,22 +6,21 @@ import {
   EmitterSubscription,
   I18nManager,
   Keyboard,
-  KeyboardEvent as RNKeyboardEvent,
   LayoutRectangle,
   NativeEventSubscription,
   Platform,
+  Pressable,
+  KeyboardEvent as RNKeyboardEvent,
   ScrollView,
   ScrollViewProps,
   StyleProp,
   StyleSheet,
   View,
   ViewStyle,
-  Pressable,
 } from 'react-native';
 
 import { useSafeAreaInsets } from 'react-native-safe-area-context';
 
-import MenuItem from './MenuItem';
 import { useInternalTheme } from '../../core/theming';
 import type { MD3Elevation, ThemeProp } from '../../types';
 import { ElevationLevels } from '../../types';
@@ -29,6 +28,7 @@ import { addEventListener } from '../../utils/addEventListener';
 import { BackHandler } from '../../utils/BackHandler/BackHandler';
 import Portal from '../Portal/Portal';
 import Surface from '../Surface';
+import MenuItem from './MenuItem';
 
 export type Props = {
   /**
@@ -360,10 +360,8 @@ const Menu = ({
         useNativeDriver: true,
       }),
     ]).start(({ finished }) => {
-      if (finished) {
-        focusFirstDOMNode(menuRef.current);
-        prevRendered.current = true;
-      }
+      focusFirstDOMNode(menuRef.current);
+      prevRendered.current = true;
     });
   }, [anchor, attachListeners, measureAnchorLayout, theme]);
 
@@ -378,12 +376,10 @@ const Menu = ({
       easing: EASING,
       useNativeDriver: true,
     }).start(({ finished }) => {
-      if (finished) {
         setMenuLayout({ width: 0, height: 0 });
         setRendered(false);
         prevRendered.current = false;
         focusFirstDOMNode(anchorRef.current);
-      }
     });
   }, [removeListeners, theme]);
 
